src/daemon/start_nfs.sh

```diff
@@ -33,6 +38,8 @@ function start_nfs {
   # create ganesha log directory since the package does not create it
   mkdir -p /var/log/ganesha/
 
+  start_dbus
```

Comment:
could you move this just after the start_rpc call ?

Reply:
non maggle

Reply:
:-1: